packages/docs/stories/Prerequisites.mdx

```diff
@@ -0,0 +1,124 @@
+import { Meta, Markdown } from "@storybook/blocks";
+import Contributing from '../../../CONTRIBUTING.md?raw';
+
+<Meta title="Prerequisites" />
+
+# Prerequisites
+
+## Fonts
+
+The Synergy Design System makes use of the `Open Sans` typeface.
+Because there are various ways of loading fonts, depending on used bundler, pre- and postprocessors and usage of a CDN, Synergy does **not provide** this font.
+You will need to include it in your project.
+
+The following information may be helpful to get you started including your own copy of `Open Sans` in your project for quicker bootstrapping:
+
+### Local Installation
+
+1. Download the `Open Sans` font from the [SICK Brand Portal](https://brand.sick.com/document/49#/-/resources-1).
```

Comment:
Why not referencing the fonts on GitHub (packages/docs/public/OpenSans) ?
I'd prefer not to reference servers we don't have control over.

Reply:
As the brand portal is the single source of truth, we will leave the comment as discussed.

Reply:
Google also only provides .ttf files in the official GitHub

Reply:
Another valid point. I think there was the possibility to load the `woff` files somewhere, but I did not find it yesterday, too.

Reply:
Yes, I also can't remember where I have downloaded the woff files initially.
Maybe buried somwhere in the older version of Google Fonts.